iotdb-client/cli/src/main/java/org/apache/iotdb/tool/ImportTsFile.java

```diff
@@ -371,9 +402,65 @@ public static void addNoResourceOrModsToQueue() throws InterruptedException {
   }
 
   public static void asyncImportTsFiles() {
+    if (isRemoteLoad) {
+      loadRemotely();
+    } else {
+      loadLocally();
+    }
+  }
+
+  public static void loadRemotely() {
+    final List<Thread> list = new ArrayList<>(threadNum);
+    for (int i = 0; i < threadNum; i++) {
+      final Thread thread = new Thread(() -> importTsFileRemotely(getRemoteLoadTsFile()));
+      thread.start();
+      list.add(thread);
+    }
+    list.forEach(
+        thread -> {
+          try {
+            thread.join();
+          } catch (final InterruptedException e) {
+            Thread.currentThread().interrupt();
+            ioTPrinter.println("importTsFile thread join interrupted: " + e.getMessage());
```

Comment:
If a worker thread is interrupted, does the main thread need to be interrupted? If so, do other worker threads also need to be interrupted?